torchvision/prototype/features/_keypoint.py

```diff
@@ -0,0 +1,67 @@
+from typing import Dict, Any, Tuple, Optional, Sequence, Union, Collection
+
+import torch
+from torchvision.prototype.utils._internal import StrEnum
+
+from ._feature import Feature, DEFAULT
+
+
+class KeypointSymmetry(StrEnum):
+    VERTICAL = "vertical"
+    HORIZONTAL = "horizontal"
+
+
+class Keypoint(Feature):
+    image_size: Tuple[int, int]
+    descriptions: Sequence[Sequence[str]]
+    symmetries: Sequence[Tuple[KeypointSymmetry, Tuple[int, int]]]
+
+    @classmethod
+    def _to_tensor(cls, data, *, dtype, device):
+        tensor = torch.as_tensor(data, dtype=dtype, device=device)
+        if tensor.shape[-1] != 2:
+            raise ValueError
+        if tensor.ndim == 1:
+            tensor = tensor.view(1, -1)
+        return tensor
+
+    @classmethod
+    def _parse_meta_data(
+        cls,
+        image_size: Tuple[int, int] = DEFAULT,  # type: ignore[assignment]
```

Comment:
This is not a common idiom in TorchVision, as `None` is favoured. Do you plan to remove this?